### PR TITLE
Explicitly context

### PR DESCRIPTION
**Pull request by brijeshnec**

### CODE REVIEW

File: Commons/src/main/java/eu/neclab/ngsildbroker/commons/tools/HttpUtils.java [hunk 1]
```diff
@@ -527,112 +526,114 @@ public static Uni<Tuple2<Object, List<Tuple2<String, String>>>> generateCompacte
...
+							Object finalCompacted;
+							if (contextHeader.isEmpty()) {
+								contextHeader.add(((List<Object>) bodyContext).get(0));
+							}
+							if (compacted.containsKey(JsonLdConsts.GRAPH)) {
+								finalCompacted = compacted.get(JsonLdConsts.GRAPH);
+							} else {
+								finalCompacted = compacted;
+							}
+							if (options != null && options.contains(NGSIConstants.QUERY_PARAMETER_CONCISE_VALUE)) {
```
Comment (amitrnec) [inline Commons/src/main/java/eu/neclab/ngsildbroker/commons/tools/HttpUtils.java:543]: Please verify concise and normalized functionality.

File: SubscriptionManager/src/main/java/eu/neclab/ngsildbroker/subscriptionmanager/controller/SubscriptionController.java [hunk 2]
```diff
@@ -23,6 +16,14 @@
 import jakarta.ws.rs.Path;
 import jakarta.ws.rs.PathParam;
 import jakarta.ws.rs.QueryParam;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.jboss.resteasy.reactive.RestResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
```
Comment (amitrnec) [inline SubscriptionManager/src/main/java/eu/neclab/ngsildbroker/subscriptionmanager/controller/SubscriptionController.java:23]: Remove space and formatting

File: Commons/src/main/java/eu/neclab/ngsildbroker/commons/tools/HttpUtils.java (hunk 1 above)
Comment (vivekNEC) [reply Commons/src/main/java/eu/neclab/ngsildbroker/commons/tools/HttpUtils.java:543]: done

File: SubscriptionManager/src/main/java/eu/neclab/ngsildbroker/subscriptionmanager/controller/SubscriptionController.java (hunk 2 above)
Comment (vivekNEC) [reply SubscriptionManager/src/main/java/eu/neclab/ngsildbroker/subscriptionmanager/controller/SubscriptionController.java:23]: done

**Pull request merged**